ui/app/selectors/selectors.js

```diff
@@ -199,8 +199,8 @@ export function accountsWithSendEtherInfoSelector (state) {
   const accounts = getMetaMaskAccounts(state)
   const { identities } = state.metamask
 
-  const accountsWithSendEtherInfo = Object.entries(accounts).map(([key, account]) => {
-    return Object.assign({}, account, identities[key])
+  const accountsWithSendEtherInfo = Object.entries(identities).map(([key, identity]) => {
+    return Object.assign({}, identity, accounts[key])
```

Comment:
It looks like the contents of `accounts[key]` are now overwriting `identity`, whereas beforehand it was the other way around.

I'm not sure it makes a difference though 🤔 As the only key they have in common is `address`, which hopefully should match.

Reply:
```suggestion
    return Object.assign({}, accounts[key], identity)
```

Reply:
I committed the inline suggestion above to avoid any unforeseen issues with this.